src/distributedTracingManager.ts

```diff
@@ -101,6 +100,7 @@ export class DistributedTracingManager extends BaseExplorer {
             title: `Update Distributed Tracing Setting`,
             location: vscode.ProgressLocation.Notification,
         }, async () => {
+            this._outputChannel.show();
             try {
                 const result = await this.updateDeviceTwin(mode, samplingRate, iotHubConnectionString, deviceIds);
```

Comment:
updateDeviceTwin may takes long time. We only want to outputChannel.show() just before this.outputLine

Reply:
Yes, that's right, however, if we put outputChannel.show() before outputLine, then we need to added it separately in try and catch block